core/types/tx_bls_test.go

```diff
@@ -10,40 +10,22 @@ import (
 	"github.com/holiman/uint256"
 )
 
-// key holds the BLS PublicKey and SecretKey for helper
-// functions defined below
-type key struct {
```

Comment:
we don't need this struct anymore since we can use `bls.PublicFromSecretKey` to derive the public key from a given secret key